wodles/gcloud/gcloud.py

```diff
@@ -77,33 +96,32 @@ def main():
             if not arguments.bucket_name:
                 raise exceptions.GCloudError(1103)
 
+            # Set arguments
             f_kwargs = {"bucket_name": arguments.bucket_name,
                         "prefix": arguments.prefix,
                         "delete_file": arguments.delete_file,
                         "only_logs_after": arguments.only_logs_after,
                         "reparse": arguments.reparse}
-            integration = GCSAccessLogs(arguments.credentials_file, logger, **f_kwargs)
+
+            # Get Access bucket logs
+            integration = GCSAccessLogs(arguments.credentials_file, gcp_logger, **f_kwargs)
             integration.check_permissions()
             num_processed_messages = integration.process_data()
 
         else:
             raise exceptions.GCloudError(1002, integration_type=arguments.integration_type)
 
     except exceptions.WazuhIntegrationException as gcloud_exception:
-        logging_func = logger.critical if \
-            isinstance(gcloud_exception, exceptions.WazuhIntegrationInternalError) else \
-            logger.error
-
-        logging_func(f'An exception happened while running the wodle: {gcloud_exception}', exc_info=log_level == 1)
+        gcp_logger.error(f'An exception happened while running the wodle: {gcloud_exception}')
         exit(gcloud_exception.errcode)
 
     except Exception as e:
-        logger.critical(f'Unknown error: {e}', exc_info=True)
```

Comment:
The `exc_info` parameter defines if additional information regarding the exception should be included. We should analyze if this should be added for the `WazuhCloudLogger.error` method.

Reply:
Included